### DISCUSSION TITLE
服务器内存占用异常后,查找数据异常，提示分桶不存在

### DISCUSSION THREAD

**Discussion by meitao20** [apache/doris#38488] [RFC]
查找表数据
SELECT * from HisDataStore LIMIT 10
报错如下
[CANCELLED]failed to get tablet: 15783651

SHOW TABLET 15783651 
信息如下
![image](https://github.com/user-attachments/assets/cc6822bc-72e6-4ee2-9a11-52752ce4b784)

该分桶对应的表id是15780345
SHOW TABLE 15780345
![image](https://github.com/user-attachments/assets/42b39bd6-5c68-4fe2-9ea0-82adc6602354)

不知问题出在哪里

**Comment by meitao20** [comment]
根据detailcmd
SHOW PROC '/dbs/10159/15780345/partitions/15783650/15780346/15783651';
获取到
ReplicaId	BackendId	Version	LstSuccessVersion	LstFailedVersion	LstFailedTime	SchemaHash	LocalDataSize	RemoteDataSize	RowCount	State	IsBad	IsUserDrop	VersionCount	PathHash	Path	MetaUrl	CompactionStatus	CooldownReplicaId	CooldownMetaId	QueryHits
15783652	6582487	13163	13163	-1		-1	0	0	0	NORMAL	false	false	-1	-1		http://192.168.1.221:8040/api/meta/header/15783651	http://192.168.1.221:8040/api/compaction/show?tablet_id=15783651	-1		0
url结果如下
![image](https://github.com/user-attachments/assets/0cac65b6-fbc9-436b-be6d-b06df1b0c615)

 
![image](https://github.com/user-attachments/assets/48722bb9-94a1-42dd-820c-77c0d36f9c2f)

**Comment by ixzc** [comment]
中文问题去ask论坛提问一下
https://ask.selectdb.com/
